Added changes to be compatible with openai > 1.0.0 package updates.

sudhirn-anyscale opened the pull request:
Updated 3 files to be compatible with openai > 1.0.0 package

Commit:
Added changes to be compatible with openai > 1.0.0 package updates.

kylehh commented on combine_anyscale_openai.ipynb:
```diff
@@ -31,6 +31,7 @@
    "source": [
     "# openai < 1.0\n",
     "import openai\n",
+    "openai.organization = \"org-xxxxxxxxxxxxx\" \n",
```
I don't think we support openai organization parameter

sudhirn-anyscale replied:
openai python package supports "organization" attribute and it works for OpenAI calls. I do not have that tested for making calls to Anyscale Endpoints

kylehh commented on streaming_chatbot.ipynb:
```diff
@@ -50,27 +52,28 @@
+    "        self.update_message_history(input)      \n",
+    "        \n",
+    "        openai.base_url = \"https://api.endpoints.anyscale.com/v1\" +'/'\n",
+    "        openai.api_key = ANYSCALE_ENDPOINT_TOKEN\n",
```
Let's create an OAI client with base_url and api_key instead of directly call openai.base_url

sudhirn-anyscale replied:
Yes, I made changes as suggested by creating a client first.

kylehh commented on streaming_chatbot.ipynb:
```diff
@@ -94,15 +97,13 @@
     "import sys\n",
     "\n",
-    "ANYSCALE_ENDPOINT_TOKEN = \"YOUR_ANYSCALE_ENDPOINT_TOKEN\"\n",
+    "ANYSCALE_ENDPOINT_TOKEN = \"esecret_xxxxxxxxxxxxxxxxxx\"\n",
```
Please change this to "YOUR_ANYSCALE_ENDPOINT_TOKEN"

sudhirn-anyscale replied:
I made changes to all scripts to leverage and import a .env file which holds secrets/passwords. Then added the file to .gitignore

sudhirn-anyscale replied:
I made changes to all scripts to leverage and import a .env file which holds secrets/passwords. Then added the file to .gitignore

kylehh commented on streaming_chatbot_langchain.ipynb:
```diff
@@ -121,7 +125,7 @@
     "\n",
     "INPUTMARKER_END = \"-- END --\"\n",
-    "ANYSCALE_ENDPOINT_TOKEN = \"YOUR_ANYSCALE_ENDPOINT_TOKEN\"\n",
+    "ANYSCALE_ENDPOINT_TOKEN = \"esecret_xxxxxxxxxxxxxx\"\n",
```
Please change this to "YOUR_ANYSCALE_ENDPOINT_TOKEN"

sudhirn-anyscale replied:
I made changes to all scripts to leverage and import a .env file which holds secrets/passwords. Then added the file to .gitignore

kylehh commented on streaming_chatbot_langchain.ipynb:
```diff
@@ -18,7 +18,11 @@
    "source": [
-    "!pip3 install -q openai langchain"
+    "#!pip3 install -q openai langchain\n",
+    "# openai > 1.3.6 requires a langchain 0.0.342 - \n",
```
Please add version to the pip install. like "pip3 install openai>1.3.6" etc

sudhirn-anyscale replied:
Done

kylehh commented on streaming_chatbot.ipynb:
```diff
@@ -17,22 +17,24 @@
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```
plz clean all output if not needed

sudhirn-anyscale replied:
cleared all outputs after testing python code snippets

Commit:
1. Code refactoring to follow best practices - first create a openai client and then use client.
2. Reactor to include .env files for secrets
3. Added function_call and Llama_guard examples

kylehh commented:
Lgtm. One small suggestion, let's rename the notebook file names to Basic_ prefix for these basic llama and function call examples

Commit:
files named to include prefix "Basic_"

sudhirn-anyscale commented:
@kylehh - renamed files as per suggestion.

kylehh reviewed (approved):
lgtm
